.github/dependabot.yml

```diff
@@ -39,7 +39,7 @@ updates:
       interval: "monthly"
     ignore:
       - dependency-name: "ubuntu"
-        versions: ["24.x", ">= 23.0.0", "23.x"]
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
 
   - package-ecosystem: "docker"
```

Comment:
You don't need to 2 entries for docker now. We had 2 because they had different ignore conditions.